openshift/api.app.yaml

```diff
@@ -0,0 +1,119 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: api
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      service: api
+    name: api
+  spec:
+    replicas: 1
+    selector:
+      service: api
+    strategy:
+      type: Rolling
```

Comment:
In prod-preview, we have a different configuration for this strategy:
```    
rollingParams:
      updatePeriodSeconds: 1
      intervalSeconds: 1
      timeoutSeconds: 600
      maxUnavailable: 25%
      maxSurge: 25%```
Shouldn't be similar ?